# Handle current state
uint16 trigger
uint8 joy1
uint8 joy2
bool button_joy
bool button_push

